core/src/main/java/org/apache/iceberg/rest/ResourcePaths.java

```diff
@@ -125,4 +125,38 @@ public String view(TableIdentifier ident) {
   public String renameView() {
     return SLASH.join("v1", prefix, "views", "rename");
   }
+
+  public String planTableScan(TableIdentifier ident) {
```

Comment:
I believe all places that use these new endpoints need to do an `Endpoint.check(...)` as it's still possible that some of these endpoints don't exist, even if the `rest-server-planning-enabled` was set by a client